rdr_service/offline/update_ehr_status.py

```diff
@@ -79,7 +88,17 @@ def update_participant_summaries_from_job(job):
                     'ehr_receipt': rec.ehrReceiptTime if rec.ehrReceiptTime else now,
                     'ehr_update': now}
             } for rec in records]
-            dispatch_participant_rebuild_tasks(patch_data, batch_size=batch_size)
+            try:
+                dispatch_participant_rebuild_tasks(patch_data, batch_size=batch_size)
+
+            except BadGateway as e:
+                LOG.error(f'Bad Gateway: {e}')
+
+            except InternalServerError as e:
+                LOG.error(f'Internal Server Error: {e}')
+
+            except HTTPException as e:
+                LOG.error(f'HTTP Exception: {e}')
```

Comment:
What do you think of passing `exc_info=True` to the logs so they’ll print out full stack traces too? (https://docs.python.org/3/library/logging.html#logging.Logger.debug)

Reply:
Yes, I will add that to the logs as well.